app/ui/common/checkbox.tsx

```diff
@@ -0,0 +1,57 @@
+export enum CheckboxColors {
```

Comment:
suggestion: you can reuse the enum ButtonColors in `app/ui/common/buttons.tsx`

Reply:
Yes good suggestion. I have create a type Color that is used each time it is needed.